kick/livestream.py

```diff
@@ -229,3 +229,30 @@ def __eq__(self, other: object) -> bool:
 
     def __repr__(self) -> str:
         return f"<Livestream id={self.id} title={self.title} streamer={self.slug}>"
+
+class Endstream:
```

Comment:
1. This should be in PastalCase
2. Throughout the rest of the library, a livestream is referred to as a "livestream" and not a "stream", this class should conform to the same standard. Ex: "LivestreamEnd"